pkgs/misc/sagetex/default.nix

```diff
@@ -4,10 +4,11 @@
 , texlive
 }:
 
-stdenv.mkDerivation rec {
+lib.fix (self: stdenv.mkDerivation rec {
   pname = "sagetex";
   version = "3.6.1";
   passthru.tlType = "run";
+  passthru.pkgs = [ self ];
```

Comment:
same and passthru is usually placed before meta

Reply:
done this as well